.azure-pipelines/pipelines.yml

```diff
@@ -635,50 +635,40 @@ stages:
     pool:
       vmImage: "ubuntu-20.04"
     steps:
+    # TODO(phlax): Move this to `env` stage and improve conditions.
     - bash: |
         VERSION_DEV="$(cat VERSION.txt | cut -d- -f2)"
         if [[ $VERSION_DEV != "dev" ]]; then
             echo "##vso[task.setvariable variable=isDev]false"
         fi
       displayName: Set development/release env
 
-    - script: ./ci/run_envoy_docker.sh './ci/do_ci.sh publish'
-      displayName: "Publish release to Github"
-      workingDirectory: $(Build.SourcesDirectory)
-      condition: eq(variables['isDev'], 'false')
-      env:
-        AZP_BRANCH: $(Build.SourceBranch)
-        GITHUB_TOKEN: $(GitHubPublicRepoOnlyAccessToken)
-
-  - job: assets
-    displayName: Publish release assets
-    condition: and(succeeded(), eq(variables['PostSubmit'], true), ne(variables['NoSync'], true), eq(variables['isTaggedRelease'], 'true'))
```

Comment:
removing the `isTaggedRelease` condition here should be ok as the (`github`) job is now set to `eq(variables['isStableBranch'], 'true')` and all of the steps check for non-dev